xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/doc/XWikiDocument.java

```diff
@@ -1241,6 +1243,31 @@ public String getRenderedContent(boolean transformationContextIsolated, XWikiCon
         return getRenderedContent(getOutputSyntax(), transformationContextIsolated, context);
     }
 
+    /**
+     * Parse and render inline contents.
+     *
+     * @param content The content to render.
```

Comment:
We usually don't end the parameter JavaDoc with a dot because it's not a sentence.